### PR TITLE
Upgrade to ocamlformat 0.26.0

### PR DESCRIPTION
**Pull request by Julow**
The classes are formatted completely differently, which might be problematic.

The aim of this commit is to gather feedback.

Changelog can be found here: https://github.com/ocaml-ppx/ocamlformat/blob/main/CHANGES.md

### CODE REVIEW

File: compiler/lib/js_output.ml [hunk 1]
```diff
@@ -856,35 +856,35 @@ struct
...
-               PP.string f "async";
-               PP.space f
-           | { async = true; generator = true } ->
-               PP.string f "async*";
-               PP.space f);
-           name f n
-         in
-         function_declaration f "" fpn (Some ()) l b loc'
+   fun (type a) f (name : PP.t -> a -> unit) (n : a) (m : method_) ->
+    match m with
```
Comment (hhugo) [inline compiler/lib/js_output.ml:860]: Is the 1 char indent on purpose ?

**Review by hhugo — COMMENTED** [review]
After a very very quick look, it looks mostly fine.
The size of the diff regarding indentation in classes makes the whole diff complicated the evaluate. Anyway to split the diff in two commits ? putting the indentation of classes on iits own ?

File: compiler/lib/js_output.ml (hunk 1 above)
Comment (Julow) [reply compiler/lib/js_output.ml:860]: Yes. An indentation of 1 here was the default but due to a bug, the indentation was 2 when there were a type annotation. This is a consistency change.

**Comment by Julow** [comment]
I splitted the changes to classes into a new commit. This commit is best viewed with `git diff -w`.

**Comment by hhugo** [comment]
Thanks a lot. LGTM

**Comment by Julow** [comment]
Thanks for the feedback. Let's not merge this until OCamlformat is released.

**Comment by Julow** [comment]
OCamlformat 0.26.0 has been released! This PR can be merged (not squashed due to `.git-blame-ignore-revs`) or closed if it's a bit too early to upgrade.

File: .git-blame-ignore-revs [hunk 2]
```diff
@@ -1,2 +1,3 @@
 # ocamlformat bump
-3c98cd3e0c4e1191983b602a8e38a1c92d259891
\ No newline at end of file
+3c98cd3e0c4e1191983b602a8e38a1c92d259891# Upgrade to OCamlformat 0.26.0
```
Comment (hhugo) [inline .git-blame-ignore-revs:2]: This comment seems to be on the wrong line

Comment (Julow) [reply .git-blame-ignore-revs]: Fixed.

**Pull request merged**